statsfs/otelstats/exporters.go

```diff
@@ -0,0 +1,70 @@
+package otelstats
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	mexporter "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/metric"
+
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/exporters/metric/prometheus"
+	"go.opentelemetry.io/otel/exporters/stdout"
+	"go.opentelemetry.io/otel/sdk/metric/controller/pull"
+	"go.opentelemetry.io/otel/sdk/metric/controller/push"
+)
+
+func initPrometheusExporter() error {
+	exporter, err := prometheus.InstallNewPipeline(prometheus.Config{}, pull.WithCachePeriod(time.Second*10))
+	if err != nil {
+		return fmt.Errorf("Failed to initialize Prometheus metric exporter: %v", err)
+	}
+
+	port := 2112
```

Comment:
Probably make it a const at the file level like

`const PrometheusPort = 2112`